include/revng/Pipes/Yield/Assembly.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <array>
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include "revng/Pipeline/Contract.h"
+#include "revng/Pipeline/Target.h"
+#include "revng/Pipes/FileContainer.h"
+#include "revng/Pipes/StringMapContainer.h"
+
+namespace revng::pipes {
+
+class YieldAssemblyPipe {
+public:
+  static constexpr const auto Name = "yield-assembly";
+
+public:
+  std::array<pipeline::ContractGroup, 1> getContract() const;
```

Comment:
@drblallo can you review the contract?

Reply:
the contract provided states that the input is both the binary and any quantity of isolated functions, and transforms each of them into disassembled functions. As far as understand that should be correct

Reply:
Enterprise.